tests/test_tiatoolbox.py

```diff
@@ -408,3 +418,41 @@ def test_command_line_save_tiles(_response_ndpi, _response_svs):
     )
 
     assert save_tiles_result.exit_code == 0
+
+
+# -------------------------------------------------------------------------------------
+# Stain Normalisation
+# -------------------------------------------------------------------------------------
+
+
+def test_reinhard_normalise():
+    """Test Reinhard colour normalisation."""
+    source_img = imread("tiatoolbox/tools/stainnorm/samples/source.png")
+
+    target_img = imread("tiatoolbox/tools/stainnorm/samples/target.png")
+
+    reinhard_img = imread("tiatoolbox/tools/stainnorm/samples/reinhard.png")
+
+    norm = reinhard_colour_normaliser.ReinhardColourNormaliser()
```

Comment:
Can we define the classes in a single file? e.g., we can have stainnorm.ReinhardColourNormaliser() and stainnorm.Ruifork()?

Reply:
The way of doing this that I can think of would be to create a script `tiatoolbox/tools/stainnorm.py` and then defining each stain normalisation class within there. Then we can have a stain_utils directory for the utils. What do you think?

Reply:
Yes sounds good to me.

Reply:
Modified in [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/ad1209a509e700c326ff847fa61437b264294582).